src/domain/child/form/ChildForm.tsx

```diff
@@ -121,12 +121,12 @@ const ChildForm: FunctionComponent<ChildFormProps> = ({
     >
       {({
         isSubmitting,
-        handleSubmit,
         values,
         errors,
         touched,
       }: FormikProps<ChildFormValues>) => (
-        <form onSubmit={handleSubmit} id="childForm">
+        <Form id="childForm" noValidate>
+          he
```

Comment:
Is this "he" a stray line of code?

Reply:
Yes it is 💆‍♂️ I'll add a fix tomorrow.

Thanks for pointing it out👍

Reply:
Glad I could be of help! 

I'm following the changes in the repo because I know I have a lot to learn from how you are developing it further. Congrats with browser testing coming in!!

Reply:
Nice catch! Seems like I need a pair of glasses 😒